src/prisma/schema.prisma

```diff
@@ -2795,6 +2795,7 @@ model Scout {
   displayName           String
   farcasterId           Int?                    @unique
   farcasterName         String? // every user only has one farcaster account
+  farcasterUsername     String?                 @unique
```

Comment:
actually this is farcasterName

Reply:
farcaster name is the display_name of the user profile. We decided to go with farcasterUsername only since that's the only farcaster related info we need and it replaces the farcasterName field